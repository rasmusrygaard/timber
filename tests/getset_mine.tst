get /
get //
get foo
get foo/
get /foo/
set / 123
set // 123
set /// 123
set foo/ 123
set /foo/ 213
create /test 123
get /test
set /test foo
get /test
set /test bar
get /test
create /test/a abc
get /test/a
set /test/a b
get /test/a
remove /test/a
set /test/a 10
