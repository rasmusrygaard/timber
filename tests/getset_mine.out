MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
CREATED
123
foo
bar
CREATED
abc
b
REMOVED
KEY NOT FOUND
